openlibrary/plugins/importapi/code.py

```diff
@@ -1,6 +1,7 @@
 """Open Library Import API
 """
 
+from typing import Tuple
```

Comment:
We run on Python >= 3.10 so this should not be needed?

Reply:
Whoops. Thanks for catching that! I've removed it.